### ISSUE TITLE
How to implement the one to one video call into my iOS application

### ISSUE THREAD

**Issue by SriSamVictor** [Temasys/SkylinkSDK-iOS-Sample#16]
Recently i installed the skylink sample application into my Iphone xr and ipad. i tried to connect the video call from one mobile to  another mobile but i would not connect to the another device. So please explain briefly how can i work with the video call(One to another)with your application.

**Comment by lyx501135** [comment]
Hi friend, thanks for using Skylink SDK, in order to run the sample app, you have to apply a key from Temasys website, www.temasys.io, it’s free, after you get the key and secret, fill them in the sample app, Setting view controller, and set the room name, then you are able to connect

**Comment by SriSamVictor** [comment]
Yes i already done that as you said and then i got success message on the console. It could not connect to the another device. So do i need to change any settings or any code?

I got this message on console.(Videocall video controller).
![Screen Shot 2020-02-14 at 7 44 12 PM](https://user-images.githubusercontent.com/30155214/74538524-6e0daa00-4f62-11ea-8469-93c5a21ab1c3.png)

**Comment by lyx501135** [comment]
Hi, are you using the OBJC sample app? you may try the Swift one, and in the sample app video call controller, you should check if the video audio send and receive is enabled , like this 

config.setAudioVideoSend(AudioVideoConfig_AUDIO_AND_VIDEO)
    config.setAudioVideoReceive(AudioVideoConfig_AUDIO_AND_VIDEO)

**Comment by SriSamVictor** [comment]
Yep, but our application is only used by objective-c and i will check the sample app in the swift and run on the mobile device and let you know after that please be available on monday. i will check on device and update with you...

**Comment by SriSamVictor** [comment]
Yes sender and receiver config was there in the objective c code

SKYLINKConnectionConfig *config = [SKYLINKConnectionConfig new];
    [config setAudioVideoReceiveConfig:AudioVideoConfig_AUDIO_AND_VIDEO];
    [config setAudioVideoSendConfig:AudioVideoConfig_AUDIO_AND_VIDEO];
